ui/components/DataTable.tsx

```diff
@@ -161,31 +161,28 @@ function filterText(
 
   return _.filter(rows, (row) => {
     let matches = false;
-    for (const colName in row) {
-      const value = row[colName];
 
-      const field = _.find(fields, (f) => {
-        if (typeof f.value === "string") {
-          return f.value === colName;
-        }
-
-        if (f.sortValue) {
-          return f.sortValue(row) === value;
-        }
-      });
+    fields.forEach((field) => {
+      if (!field.textSearchable) return matches;
 
-      if (!field || !field.textSearchable) {
-        continue;
+      let value;
+      if (field.sortValue) {
+        value = field.sortValue(row);
+      } else {
+        typeof field.value === "function"
```

Comment:
Can we re-write this ternary as:

```
value =
          typeof field.value === "function"
            ? field.value(row)
            : row[field.value];
```

It's a standard form of an assignment with ternary instead of having multiple assignments.